sr_ethercat_hand_config/logging/ws_diff.pl

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/perl
```

Comment:
Sadly I have to second Andriy's opinion, (although to be fair I'm not very good at reading shell script either).
As a general rule we should stick to our main languages unless there is a powerful reason not to do it.